backend/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM rust:1.89 AS base
```

Comment:
The API is one binary at the moment. Another one is the periodic-tasks. So we either make it explicit that this file refers to the API (e.g. api.dockerfile) or keep the file where it was before i.e. in the `api` directory.